dev/tests/js/jasmine/tests/lib/mage/browser.test.js

```diff
@@ -41,8 +42,8 @@ define([
                         }
                     };
                 });
-            obj.openDialog('instance/url', 100, 100, 'title', options);
-            obj.openDialog('instance/url', 100, 100, 'title', options);
+            obj.openDialog(openUrl, 100, 100, 'title', options);
+            obj.openDialog(openUrl, 100, 100, 'title', options);
             expect($.ajax.calls.count()).toBe(1);
```

Comment:
Can we verify if the pathId is set on these tests?

Reply:
actually this test will fail if in url have not current_tree_path, so they cover this case

Reply:
because they always set pathId from url, so this case is covered